database/seeders/DatabaseSeeder.php

```diff
@@ -2,18 +2,57 @@
 
 namespace Database\Seeders;
 
+use App\Models\Answer;
+use App\Models\Question;
 use Illuminate\Database\Console\Seeds\WithoutModelEvents;
 use Illuminate\Database\Seeder;
 
 class DatabaseSeeder extends Seeder
 {
+
     /**
      * Seed the application's database.
+     *public function run()
+    {
+    Question::truncate();
+    Answer::truncate();
+
+    // \App\Models\User::factory(10)->create();
+
+
+    $question = Question::create([
+    'title'=>'this is question 1 title from seeder',
+    'body' => 'this is question body from seeder',
```

Comment:
There should be no commented code only helpful comments are allowed

Reply:
Ok